Debian/8/Makefile

```diff
@@ -0,0 +1,86 @@
+include ../../VERSION
+
+IN = input
+OUT = output
+BUILD = build
+TRANS = transforms
+SHARED = ../../shared
+SHARED_OVAL = $(SHARED)/oval
+CONF = ../../config
+UTILS = utils
+DIST = dist
+
+OVAL_5_11 := $(shell oscap --version | grep -q "OVAL Version: 5.11.*"; echo $$?)
+
+ID = ssg
+PROD = debian8
+PROD_OVAL = $(BUILD)/$(PROD)_oval
+
+all: shorthand2xccdf guide content dist
+
+shorthand-guide:
+	xsltproc -o $(OUT)/$(ID)-$(PROD)-shorthand.xml $(IN)/guide.xslt $(IN)/guide.xml
+	xmllint --format --output $(OUT)/$(ID)-$(PROD)-shorthand.xml $(OUT)/$(ID)-$(PROD)-shorthand.xml
+
+shorthand2xccdf: shorthand-guide
+	xsltproc -o $(OUT)/unlinked-unresolved-$(PROD)-xccdf.xml $(TRANS)/shorthand2xccdf.xslt $(OUT)/$(ID)-$(PROD)-shorthand.xml
+	oscap xccdf resolve -o $(OUT)/unlinked-$(PROD)-xccdf.xml $(OUT)/unlinked-unresolved-$(PROD)-xccdf.xml
+
+checks:
+	# Make intermediate $(PROD_OVAL) directory to hold final list of OVAL checks for $(PROD)
+	mkdir -p $(PROD_OVAL)
+	# Search $(SHARED_OVAL) and $(IN)/oval directories to find all product specific OVAL checks,
+	# which are regular files (not symlinks). Merge the final list into $(PROD_OVAL) directory
+	find $(SHARED_OVAL) $(IN)/oval -maxdepth 1 -type f -name *.xml -exec cp {} $(PROD_OVAL) ';'
+	# If openscap on the system supports OVAL-5.11 language version, include also OVAL-5.11 checks
+	# add local Debian8 specific templated ovals
+	$(MAKE) -C $(IN)/oval/templates all
+	$(MAKE) -C $(IN)/oval/templates copy
```

Comment:
@iankko thoughts on copying from the templates during build? The rest of the content's Makefiles do not do this and all checks/remediation scripts exist in `oval` and `remediations` directories even if they were built off of templates.